DOC add dicodile-related install information (#86)
alphacsc/alphacsc: README.rst

    plt.show()

Dicodile backend
================

AlphaCSC can use a `dicodile <https://github.com/tomMoral/dicodile>`_-based backend to perform sparse encoding in parallel.

> To install Dicodile, run ``pip install alphacsc[dicodile]``.

Reviewer: ```suggestion
To install dicodile, run ``pip install alphacsc[dicodile]``.
```

Reviewer: Besides this nitpick lgtm

> OpenMPI issues
> --------------

Reviewer 2: ```suggestion
Known OpenMPI issues
--------------------
```

> When self-installing OpenMPI (for instance to run Dicodile on a single machine, or for continuous integration) it may be necessary to disable Docker-related virtual NICs, for instance by running ``export OMPI_MCA_btl_tcp_if_exclude="docker0"``.

Reviewer: ```suggestion
When self-installing OpenMPI (for instance to run `dicodile` on a single machine, or for continuous integration) it may be necessary to disable Docker-related virtual NICs, for instance by running ``export OMPI_MCA_btl_tcp_if_exclude="docker0"``.
```

> When self-installing OpenMPI (for instance to run Dicodile on a single machine, or for continuous integration) it may be necessary to disable Docker-related virtual NICs, for instance by running ``export OMPI_MCA_btl_tcp_if_exclude="docker0"``.

Reviewer 2: ```suggestion
When self-installing OpenMPI (for instance to run Dicodile on a single machine, or for continuous integration), running the `dicodile` solver might end up causing a deadlock (no output for a long time). It is often due to communication issue between the workers. This issue can often be solved by disabling Docker-related virtual NICs, for instance by running ``export OMPI_MCA_btl_tcp_if_exclude="docker0"``.
```
